hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/ingest/SubmissionManager.java

```diff
@@ -97,17 +97,16 @@ public SubmissionManager(
             @NonNull final ConfigProvider configProvider,
             @NonNull final Metrics metrics) {
         this.platform = requireNonNull(platform);
-        this.submittedTxns = requireNonNull(deduplicationCache);
```

Comment:
I think it is good practice to keep `this.` when a field is initialized in the constructor.

Reply:
Yes...my IntelliJ keeps removing those. I think I fix it and it shouldn't happen now.